torch/_functorch/functional_call.py

```diff
@@ -122,19 +121,29 @@ def compute_loss(params, x, t):
     if isinstance(parameter_and_buffer_dicts, dict):
         parameters_and_buffers = parameter_and_buffer_dicts
     elif isinstance(parameter_and_buffer_dicts, Sequence):
-        if not all(isinstance(d, dict) for d in parameter_and_buffer_dicts):
-            raise ValueError(
-                "Expected all elements of parameter_and_buffer_dicts to be dictionaries"
-            )
-        all_keys = [k for d in parameter_and_buffer_dicts for k in d.keys()]
-        repeated_keys = [key for key, n in Counter(all_keys).items() if n > 1]
+        for d in parameter_and_buffer_dicts:
+            if not isinstance(d, dict):
+                raise ValueError(
+                    "Expected all elements of parameter_and_buffer_dicts to be dictionaries"
+                )
```

Comment:
I assume the rewrites are because Dynamo doesn't support a feature. Could we file some issues for those?

Reply:
They are working now. So, all good!